### ISSUE TITLE
Still a problem with notebook cell embedding in book project/pdf output

### ISSUE THREAD

**Issue by MHellmund** [quarto-dev/quarto-cli#5255]
### Bug description

Thank you for the fast fixing of https://github.com/quarto-dev/quarto-cli/issues/5239 !  I have the next problem.

- fresh `git pull` of quarto, book project, pdf output
-  embedding of a notebook cell when the path to the notebook starts with dots: 
       `{{< embed ../notebooks/nb1.ipynb#cell1  >}}`

      - `quarto render --to html` works
      - `quarto render --to pdf` fails
```
     ERROR: Unable to embed content from notebook 'nb1.ipynb'
The file /home/hellmund/Julia/23/notebooks/nb1.ipynb doesn't exist or cannot be read.

Error: Unable to embed content from notebook 'nb1.ipynb'
The file /home/hellmund/Julia/23/notebooks/nb1.ipynb doesn't exist or cannot be read.
    at notebookIncludes (file:///home/hellmund/W/quarto-cli/src/core/jupyter/jupyter-embed.ts:203:17)
    at replaceNotebookPlaceholders (file:///home/hellmund/W/quarto-cli/src/core/jupyter/jupyter-embed.ts:208:18)
    at async renderPandoc (file:///home/hellmund/W/quarto-cli/src/command/render/render.ts:112:26)
    at async renderSingleFileBook (file:///home/hellmund/W/quarto-cli/src/project/types/book/book-render.ts:341:30)
    at async Object.onComplete (file:///home/hellmund/W/quarto-cli/src/project/types/book/book-render.ts:283:17)
    at async renderFiles (file:///home/hellmund/W/quarto-cli/src/command/render/render-files.ts:534:12)
    at async renderProject (file:///home/hellmund/W/quarto-cli/src/command/render/project.ts:263:23)
    at async Command.fn (file:///home/hellmund/W/quarto-cli/src/command/render/cmd.ts:211:26)
    at async Command.execute (file:///home/hellmund/W/quarto-cli/src/vendor/deno.land/x/cliffy@v0.25.4/command/command.ts:1790:7)
    at async quarto (file:///home/hellmund/W/quarto-cli/src/quarto.ts:122:3)
```

(A path to the notebook cell that does not start with dots works since yesterday's patch!)
Minimal example with both path variants attached.
[test2.zip](https://github.com/quarto-dev/quarto-cli/files/11287728/test2.zip)

### Checklist

- [ ] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [ ] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [ ] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [ ] Please document the operating system you're running. If on Linux, please provide the specific distribution.
- [ ] Please provide the output of `quarto check` so we know which version of quarto and its dependencies you're running.

**Comment by itayko2** [comment]
try to insert some text between 2 embeddings

**Comment by dragonstyle** [comment]
This should be fixed in the final release of 1.3 (and on main as well).
